addOns/network/src/main/java/org/zaproxy/addon/network/ExtensionNetwork.java

```diff
@@ -298,6 +298,18 @@ ConnectionOptions getConnectionOptions() {
         return connectionOptions;
     }
 
+    public HttpProxy getHttpProxy() {
```

Comment:
It would be better to expose an interface, this class was intended to be internal, otherwise move to the common package (and add a since tag).